scripts/monitoring/cron-send-elb-health.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python
+# vim: expandtab:tabstop=4:shiftwidth=4
+'''
+  Collect health of nodes behind all available ELBs
+'''
+#
+#   Copyright 2015 Red Hat Inc.
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# Disabling invalid-name because pylint doesn't like the naming conention we have.
+# pylint: disable=invalid-name
+# Disabling the import-errors
+# pylint: disable=import-error
+# Disabling line too long for one character
+# pylint: disable=line-too-long
+# pylint: disable=pointless-string-statement
+# pylint: disable=deprecated-lambda
+# pylint: disable=bad-builtin
+# pylint: disable=bare-except
+
+from ConfigParser import SafeConfigParser
+import argparse
+from openshift_tools.monitoring.metric_sender import MetricSender
+import urllib2
+import yaml
+#import boto.ec2.elb
+#import boto.utils
+import boto3
+
+# number instances behind elb
+elb_no_instances = []
+
+# number of unhealthy instances
+elb_instance_unhealthy_count = []
```

Comment:
The name of the variable is a little confusing, it names count, but it is actually an array. And a `len($var_name)` is a real count. Maybe we should rename this to something simple like `elb_unhealthy_instances`